packages/agreement-process/test/mockUtils.ts

```diff
@@ -201,3 +209,51 @@ export const authDataAndDelegationsFromRequesterIs = (
       };
     })
     .exhaustive();
+
+export const getAMockDescriptor = (state?: DescriptorState): Descriptor => ({
+  id: generateId(),
+  version: "1",
+  docs: [],
+  state: state || descriptorState.draft,
+  audience: ["pagopa.it"],
+  voucherLifespan: 60,
+  dailyCallsPerConsumer: 10,
+  dailyCallsTotal: 1000,
+  createdAt: new Date(),
+  serverUrls: ["pagopa.it"],
+  agreementApprovalPolicy: "Automatic",
+  attributes: {
+    certified: [],
+    verified: [],
+    declared: [],
+  },
+  ...(state === descriptorState.archived ? { archivedAt: new Date() } : {}),
+  ...(state === descriptorState.suspended ? { suspendedAt: new Date() } : {}),
+  ...(state === descriptorState.deprecated ? { deprecatedAt: new Date() } : {}),
+  ...(state === descriptorState.published ? { publishedAt: new Date() } : {}),
+  // rejectionReasons: [],
+});
+
+export const getAMockDescriptorPublished = (
+  descriptorId: DescriptorId = generateId<DescriptorId>(),
+  certifiedAttributes: EServiceAttribute[][] = [],
+  declaredAttributes: EServiceAttribute[][] = [],
+  verifiedAttributes: EServiceAttribute[][] = []
+): Descriptor => ({
+  ...getAMockDescriptor(descriptorState.published),
+  id: descriptorId,
+  attributes: {
+    certified: certifiedAttributes,
+    declared: declaredAttributes,
+    verified: verifiedAttributes,
+  },
+  rejectionReasons: undefined,
+});
+
+export const getAMockVerifiedTenantAttribute = (
```

Comment:
Same, we already have this in commons-test, called  `getMockVerifiedTenantAttribute`